tools/local_fix_format.sh

```diff
@@ -31,9 +31,9 @@ fi
 
 # The following condition is added to be able to run this script using Docker,
 # due to a problem to locate clang using WSL on Windows. Use with the following command:
-# BUILD_REASON=local-ci ./ci/run_envoy_docker.sh ./tools/local_fix_format.sh
+# 'BUILD_REASON=local-ci ./ci/run_envoy_docker.sh ./tools/local_fix_format.sh'
```

Comment:
to be consistent with the "-verbose" flag as well as "-main" and "-all" do you want to use a similar flag rather than an environment variable to convey this?

Also "local-ci" seems confusing since it's already local. Instead consider calling this "-docker"? This also seems like an awkward way to have to call it. Why not do something like

```
if [[ $# -gt 0 && "$1" == "-docker" ]]; then
  exec ./ci/run_envoy_docker.sh "$0" -run-build-setup "$@"
fi
if [[ $# -gt 0 && "$1" == "-docker" ]]; then
  . ci/build_setup.sh
fi
```

You'd probably want to make that the first option or you'll lose the 'verbose' flag.

Reply:
I think I understood what you meant, but had to make a slight adjustment for this to work, please let me know if that works